README.md

```diff
@@ -17,15 +17,16 @@ python3 setup.py install --user
 pip3 install standup-notes
 ```
 ### Bash Setup
-For a 1 time execution, use
+
+To have bash completion, find where the bash_completion.d file is in your system by running the following command
 ```
-source /path/to/standup-notes.bash
+pip3 show standup-notes
 ```
-To permanently have bash completion, copy the standup-notes.bash into the bash-completion.d folder in your system
+Under the ```Location: ``` tab, this is where ```standup-notes.bash``` is located.
```

Comment:
The Location directory won't be exactly where standup-notes.bash is, it's in "/standup_notes/resources/" under the Locations directory.

Reply:
Gotcha, changed the directory in readme